installer/PowerToysSetup/Product.wxs

```diff
@@ -241,6 +241,66 @@
                 </Directory>
               </Directory>
               <Directory Id="LauncherPropertiesFolder" Name="Properties" />
+              <Directory Id="LauncherruntimesFolder" Name="runtimes">
+                <Directory Id="Launcherlinux64Folder" Name="linux-x64">
+                  <Directory Id="Launcherlinux64nativeFolder" Name="native">
+                    <Directory Id="Launcherlinux64netFolder" Name="netstandard2.0">
+                    </Directory>
+                  </Directory>
+                </Directory>
+                <Directory Id="Launcherosx64Folder" Name="osx-x64">
+                  <Directory Id="Launcherosx64nativeFolder" Name="native">
+                    <Directory Id="Launcherosx64netFolder" Name="netstandard2.0">
+                    </Directory>
+                  </Directory>
+                </Directory>
+                <Directory Id="LauncherwinFolder" Name="win">
+                  <Directory Id="LauncherwinLibFolder" Name="lib">
+                    <Directory Id="LauncherwinnetFolder" Name="netstandard2.0">
+                    </Directory>
+                  </Directory>
+                </Directory>
+                <Directory Id="Launcherwin10armFolder" Name="win10-arm">
+                  <Directory Id="Launcherwin10armnativeFolder" Name="native">
+                  </Directory>
+                </Directory>
+                <Directory Id="Launcherwin10armx64Folder" Name="win10-armx64">
+                  <Directory Id="Launcherwin10armx64nativeFolder" Name="native">
+                    <Directory Id="Launcherwin10releaseFolder" Name="release">
+                    </Directory>
+                    <Directory Id="Launcherwin10debugFolder" Name="debug">
+                    </Directory>
+                  </Directory>
+                </Directory>
+                <Directory Id="Launcherwin10x64Folder" Name="win10-x64">
+                  <Directory Id="Launcherwin10x64nativeFolder" Name="native">
+                    <Directory Id="Launcherwin10x64releaseFolder" Name="release">
+                    </Directory>
+                    <Directory Id="Launcherwin10x64debugFolder" Name="debug">
```

Comment:
@alekhyareddy28 @jyuwono.  What was the error that was requiring debug.  I think this is only for the sqllite interop, but I don't know what uses that.  It doesn't make sense that we need debug assemblies, and I'm worried that this is indicative that our build is configured incorrectly.

Reply:
Description: A .NET Core application failed. Application: PowerLauncher.exe Path: C:\Program Files\PowerToys\modules\launcher\PowerLauncher.exe Message: Error: An assembly specified in the application dependencies manifest (PowerLauncher.deps.json) was
 not found: package: 'Microsoft.VCRTForwarders.140', version: '1.0.2-rc' path: 'runtimes/win10-x64/native/debug/concrt140d_app.dll'

Reply:
SQLite.Interop.dll is a type of DLL file associated with System.Data.SQLite. 
[Reference](https://www.solvusoft.com/en/files/missing-not-found-error/dll/windows/laplink-inc/pcmover-ultimate/sqlite-interop-dll/)

Reply:
I don't have the full context here, but please make sure that no partial debug options will eventually end up in master.

Reply:
I believe that assembly is brought in by the xaml islands.  We don't reference it explicitly.  I don't want to block this from getting in much longer, but can create a new issue to remove the debug and ios/android runtimes.   @enricogior does that work for you?

Reply:
I'm seeing the same issue for adding in SettingsV2 to the MSI as well. It looks like the deps.json file references ARM, x64 and x86 versions of all the dependencies and debug and release versions of each of them. The program launches only if all these referenced dlls are present (even though the ARM one for example would not be used). [This readme](https://github.com/dotnet/cli/blob/v2.0.0/Documentation/specs/runtime-configuration-file.md) has more context on that file.

Reply:
@ryanbodrug-microsoft sure that sounds good.